vividus-bdd-engine/src/main/java/org/vividus/bdd/expression/DiffDateExpressionProcessor.java

```diff
@@ -23,9 +23,12 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
+import javax.inject.Named;
+
 import org.apache.commons.lang3.StringUtils;
 import org.vividus.util.DateUtils;
 
+@Named("diffDateToExpression")
```

Comment:
actual name is not required: `@Named("diffDateToExpression")` -> `@Named`
here and below